watch.js

```diff
@@ -193,7 +204,7 @@ FirebaseWatcher.prototype.connect = function() {
 					handleReply(self, msg);
 				} else if (msg.d.a == 'd' || msg.d.a == 'm') {
 					if (self.state == IGNORE_NEXT_STREAM) {
-						self.state = NORMAL;
+						self.log('msg, ' + self.watchReqs + ' to go');
 					} else {
 						handleUpdate(self, msg.d.b.p, msg.d.b.d);
```

Comment:
This is the protocol the firebase server uses, so we're stuck with it. ¯\\\_(ツ)\_/¯

The firebase wire protocol uses these one-letter variables because websocket connections did not originally support compression (and the firebase server appears to still not support compression), so it was necessary to keep overhead as small as possible.